routes/api/bdgtRoutes.js

```diff
@@ -1,5 +1,21 @@
+const Purchase = require("../../lib/purchase");
+const User = require("../../lib/User");
 const router = require("express").Router();
 
-// TODO: Add BDGT API ROUTES 
+// TODO: Add BDGT API ROUTES
+router.post("/purchase", (req, res) => {
+  Purchase.create({
+    price: req.body.price,
+  });
 
-module.exports = router;
\ No newline at end of file
+  if (req.session.user) {
```

Comment:
should not be needed- already added middleware to catch users who aren't logged in yet 😄